src/pretix/plugins/sendmail/templates/pretixplugins/sendmail/send_form.html

```diff
@@ -57,6 +60,12 @@ <h1>
                                     {% trans "Tickets" %}
                                 </p>
                             {% endif %}
+                            {% if form.cleaned_data.attach_ical %}
+                                <p>
+                                    <span class="fa fa-file" aria-hidden="true"></span>
+                                    {% trans "Calendar invites" %}
```

Comment:
maybe we could even stick with the same label
```suggestion
                                    {% trans "Attach calendar files" %}
```

Reply:
Ah, thanks, I couldn’t find that when I created the PR.